rca/api_content/content.py

```diff
@@ -171,17 +183,8 @@ def pull_news_and_events(programme_type_slug=None):
 
     query = query.urlencode()
     blog_url = f"{settings.API_CONTENT_BASE_URL}/api/v2/pages/?{query}"
-    try:
-        response = requests.get(url=blog_url, timeout=5)
-        response.raise_for_status()
-        logger.info("Pulling Blogs from API")
-    except requests.exceptions.HTTPError:
-        error_text = "Error occured when fetching Blog data"
-        logger.exception(error_text)
-        raise CantPullFromRcaApi(error_text)
-    else:
-        data = response.json()
-        blog_data = parse_items_to_list(data, "News")
+    data = fetch_data(blog_url)
+    blog_data = parse_items_to_list(data, "News")
```

Comment:
Are "news" and "blog" being used interchangeably? If so it'll help readability if just one term is used, otherwise a comment to explain might help